test/api.mocha.coffee

```diff
@@ -606,30 +607,30 @@ describe "API", ->
           user = _user
           done()
 
-    it "Handles unsubscription", (done) ->
-      cron = ->
-        user.lastCron = moment().subtract(1, "d")
-        user.fns.cron()
-
-      expect(user.purchased.plan.customerId).to.not.be.ok()
-      payments.createSubscription user,
-        customerId: "123"
-        paymentMethod: "Stripe"
-
-      expect(user.purchased.plan.customerId).to.be.ok()
-      shared.wrap user
-      cron()
-      expect(user.purchased.plan.customerId).to.be.ok()
-      payments.cancelSubscription user
-      cron()
-      expect(user.purchased.plan.customerId).to.be.ok()
-      expect(user.purchased.plan.dateTerminated).to.be.ok()
-      user.purchased.plan.dateTerminated = moment().subtract(2, "d")
-      cron()
-      expect(user.purchased.plan.customerId).to.not.be.ok()
-      payments.createSubscription user,
-        customerId: "123"
-        paymentMethod: "Stripe"
-
-      expect(user.purchased.plan.dateTerminated).to.not.be.ok()
-      done()
\ No newline at end of file
+      it "Handles unsubscription", (done) ->
```

Comment:
Just needed to be indented to be in the correct block (note: Block set to skip right now)